include/fuzzy/costs.hh

```diff
@@ -4,18 +4,58 @@
 
 namespace fuzzy
 {
+  struct EditCosts {
+    const float _insert;
+    const float _delete;
+    const float _replace;
```

Comment:
These attributes are public so you can remove the leading underscore.

Reply:
Keyword "delete" cannot be used for a variable. Is "delete_" ok as a variable name? Or do you have other suggestions.

Reply:
Right, then you can use the same names as the constructor arguments: `insert_cost`, `delete_cost`, and `replace_cost`.